docs/working-with-js-objects-and-values.md

```diff
@@ -252,32 +263,78 @@ In the route function, the `_type` argument starts with an underscore. When
 binding to JavaScript objects with fields that are reserved keywords in OCaml,
 Melange allows the use of an underscore prefix for the labeled arguments. The
 resulting JavaScript object will have the underscore removed from the field
-names. This is only required for the `mel.obj` attribute, while for other cases,
-the `mel.as` attribute can be used to rename fields.
+names. This is only required for the `@mel.obj` attribute, while for other
+cases, the `@mel.as` attribute can be used to rename fields.
```

Comment:
Same comments as above for inclusion of `@` and OCaml syntax.

Reply:
Fixed in another PR: https://github.com/melange-re/melange-re.github.io/pull/212/commits/ba9bf1d433289f0823397bc4db0c128f02e8d6c3